x/harvest/keeper/deposit.go

```diff
@@ -9,26 +9,21 @@ import (
 )
 
 // Deposit deposit
-func (k Keeper) Deposit(ctx sdk.Context, depositor sdk.AccAddress, amount sdk.Coin, depositType types.DepositType) error {
+func (k Keeper) Deposit(ctx sdk.Context, depositor sdk.AccAddress, amount sdk.Coin) error {
 
-	err := k.ValidateDeposit(ctx, amount, depositType)
+	err := k.ValidateDeposit(ctx, amount)
 	if err != nil {
 		return err
 	}
 
-	switch depositType {
-	case types.LP:
-		err = k.supplyKeeper.SendCoinsFromAccountToModule(ctx, depositor, types.ModuleAccountName, sdk.NewCoins(amount))
-	default:
-		return sdkerrors.Wrap(types.ErrInvalidDepositType, string(depositType))
-	}
+	err = k.supplyKeeper.SendCoinsFromAccountToModule(ctx, depositor, types.ModuleAccountName, sdk.NewCoins(amount))
 	if err != nil {
 		return err
 	}
 
-	deposit, found := k.GetDeposit(ctx, depositor, amount.Denom, depositType)
+	deposit, found := k.GetDeposit(ctx, depositor, amount.Denom)
```

Comment:
Add an `IncrementTotalDeposited` call here

Reply:
Created an Asana card for this